crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -506,17 +516,19 @@ impl ArtifactFetcher {
         }
 
         // Otherwise, fall back to scraping from the Web.
-        if let Some(url) = key.abs_path() {
-            // TODO: Do we want to scope this to the project/org?
-            let scraped_file = self
-                .sourcefiles_cache
-                .fetch_public_url(url.to_owned())
-                .await;
-
-            return scraped_file.map(|contents| CachedFile {
-                contents,
-                sourcemap_url: None,
-            });
+        if self.allow_scraping {
+            if let Some(url) = key.abs_path() {
+                // TODO: Do we want to scope this to the project/org?
```

Comment:
Not really clear on what needs to be done about this comment tbh

Reply:
We would need to thread through the incoming `scope` from the request through all the processing. Should bo doable as a followup